cmd/runner/token.go

```diff
@@ -0,0 +1,81 @@
+package runner
+
+import (
+	"os"
+	"time"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/spf13/cobra"
+
+	"github.com/CircleCI-Public/circleci-cli/api/runner"
+)
+
+func newTokenCommand(r *runner.Runner, preRunE validator) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "token",
+		Short: "Operate on runner tokens",
+	}
+
+	createOpts := struct {
+		config bool
+	}{}
+	createCmd := &cobra.Command{
+		Use:     "create <resource-class> <nickname>",
+		Short:   "Create a token for a resource-class",
+		Args:    cobra.ExactArgs(2),
+		PreRunE: preRunE,
+		RunE: func(_ *cobra.Command, args []string) error {
+			token, err := r.CreateToken(args[0], args[1])
+			if err != nil {
+				return err
+			}
+
+			if createOpts.config {
```

Comment:
This is a great idea!

Maybe it should be on by default?

Reply:
🤔